eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -327,5 +333,14 @@ function Remove-EmptySections {
     }
     $ChangeLogEntry.ReleaseContent = $sanitizedReleaseContent.ToArray()
   }
-  return $changeLogEntry
+
+  if ($changeLogEntries -and $Version)
+  {
+    $ChangeLogEntries[$Version] = $ChangeLogEntry
```

Comment:
This is redundant because you are modifying this object by reference. I don't think based on reading the outer code that this will cause problems, but writing and using the function in this way suggests the object is being changed by value, so there's a small chance it could lead to bugs.